packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -14,13 +15,13 @@ class StatsGroup extends GroupHelper {
 
     resolveOptions() {
         if (this.args.verbose && this.args.stats) {
-            logger.warn('Conflict between "verbose" and "stats" options. Using verbose.');
+            logger.warn(yellow('Conflict between "verbose" and "stats" options. Using verbose.'));
```

Comment:
I think wepback logger should colors errors and warnings out of box, we don't need do this

Also `stats` has `color` option, what we should respect

Reply:
> I think wepback logger should colors errors and warnings out of box, we don't need do this

I think the same, should I create PR at the core ?

> Also stats has the color option, what we should respect

I think we should enable color output by default. Users can disable output if needed with `--no-stats-colors`.

Reply:
@snitin315 

> I think the same, should I create PR at the core ?

yes, it is good idea

> I think we should enable color output by default. Users can disable output if needed with --no-stats-colors.

I think we can use `https://github.com/jorgebucaran/colorette` instead of `chalk`, it is faster, less and has automatic color support detection

Reply:
https://github.com/jorgebucaran/colorette looks good to me, I think we can use it. 
/cc @webpack/cli-team

Reply:
`chalk` is already a dependency in `webpack-cli`. For the scope of this PR I would recommend either to use chalk only, or first migrate the chalk to `colorette` and then continue with this PR.

Reply:
> first migrate the chalk to colorette

Yes think it is better to first

Reply:
We have now migrated to `colorette` and the runtime logger we are using is meant for browsers and not for Node.js. So we have to do it like this `logger.warn(yellow('warning'))` until webpack exposes a runtime logger API for Node. 

Refer https://github.com/webpack/webpack/issues/11215#issuecomment-667217564

Reply:
Yes, let's use `console` for logging while webpack doesn't implement node logger